isic_challenge_scoring/classification.py

```diff
@@ -31,7 +31,7 @@ def __init__(
         truth_probabilities: pd.DataFrame,
         prediction_probabilities: pd.DataFrame,
         truth_weights: pd.DataFrame,
-        validation_metric: Optional[ValidationMetric] = None,
+        target_metric: ValidationMetric = ValidationMetric.BALANCED_ACCURACY,
```

Comment:
```suggestion
        target_metric: ValidationMetric,
```

I'm fine if the internal API makes this mandatory. A default might be a source of bugs.